### PR TITLE
Cucumber feature subdirs

### PR DESCRIPTION
**Pull request by jphpsf**
I re-organized the features into sub-directories. This is a tip I read in the Rspec book which I find useful. As a project grows, it makes it easier to navigate through the features.

### CODE REVIEW

**Comment by radar** [comment]
This is quite a large change to make to the application and the book this far along.

However, I do agree with you on this. Don't think that I am ignoring this pull request, just rather letting it marinate while I come up with the motivation to fix this in the book as well as in ticketee.

**Comment by jphpsf** [comment]
No worries.

It is true that it is a big change that has impact on the book as you would have to change every listing that references the features path and add the subdir to it.

Maybe you can mention the idea in a tip box (like the one about "Combining redirect_to flash" on page 86)?

That would be a lot less changes to do in the book :)

**Comment by radar** [comment]
This works fine for running the entire feature set but when attempting to run a single feature using `bundle exec cucumber` it claims that it cannot find _any_ of the steps.

Did you try this?

**Comment by jphpsf** [comment]
I did not try this. Out of habit, I just run `cucumber` and it works.

I tried `bundle exec cucumber` as well as `rake cucumber` and they both give the same issue

I will investigate and get back to you

**Comment by jphpsf** [comment]
So after digging a little, it seems that the `config/cucumber.yml` file needs a minor change: changing the line `default: <%= std_opts %>` to `default: <%= std_opts %> features` works.

Without that, it does not load the code under support (you can see that with --verbose).

I'll do more testing and update the pull request

Cheers

**Comment by jphpsf** [comment]
Here are my observations:

1) The commands `cucumber` and `bundle exec cucumber` are both working

2) The command `rake cucumber` does not work unless you make the change I mentioned in my previous comment (adding `features` to the default options; it looks like the cucumber.yml is outdated in the ticketee project as recent version of cucumber-rails would produce the cucumber.yml with that `default: <%= std_opts %> features` line)

3) The commands `cucumber features/projects/creating.feature` or `bundle exec cucumber features/projects/creating.feature` do not work unless you pass the option `--require feature`. This is getting a lot to type, so one could create a shell alias or function for that. Or even a script. The issue seemed to have come up several time on cucumber mailing list (see for instance http://groups.google.com/group/cukes/browse_thread/thread/b9cccb18069b4a07)

TL;DR: If you want to target a specific feature, you need to have a `--require features`

Cheers
